src/news/2021-07-13-Web-Site-Starter-Kits.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Web Site Starter Kits
+filename: Web-Site-Starter-Kits
+date: '2021-07-13'
+---
+The FLOE Team is developing starter kits to accelerate the development of web sites; providing a solid foundation to
+build from and maintain over time.
+
+[Trivet](https://github.com/fluid-project/trivet) is a simple starter kit for using [Eleventy](https://www.11ty.dev) to
+create accessible, localizable static sites.
```

Comment:
Might be worth making a note about the relationship between the Floe website and Trivet. Maybe say "The Floe website is an example of a site generated using Trivet."?

Although this isn't entirely accurate as the Floe site came first then abstracted into Trivet. Maybe @greatislander or @BlueSlug has some idea how to phrase this?

Reply:
I think that's the case. There are other sites I think, like the guides and the handbook, that could be referenced. Is there any others that might be better or also appropriate?

Reply:
The Co-Design kit was made with Trivet but it's not really launched yet.